Lib/test/test_uuid.py

```diff
@@ -21,7 +21,7 @@ def importable(name):
     try:
         __import__(name)
         return True
-    except:
+    except ModuleNotFoundError:
```

Comment:
If you want to be maximally pedantic you can go even more specific:

```suggestion
    except ModuleNotFoundError as e:
        if e.name == name:
            return False
        raise
```

Eg if a module is successfully found but then imports another module which isn't found, this will raise but the code in the PR will not

Reply:
There's only one other place in the repo that looks at `e.name` so this seems to be a pretty uncommon pattern and maybe not worth for test code? Though I don't have a strong preference either way :)

Reply:
Yeah I'm cool with it either way

Reply:
Alright, I'll keep it as is for now and let's see what other reviewers think :)